.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 .vscode
 .env
-.DS_Store
\ No newline at end of file
+.DS_Store
+.idea/
```

Comment:
I think we should remove these (and `.vscode` too): they belong to local / OS configuration (`~/.gitignore`) and are not related to the repository.